source/NVDAObjects/IAccessible/MSHTML.py

```diff
@@ -911,13 +920,22 @@ def _get_rowCount(self):
 		except:
 			raise NotImplementedError
 
-	def scrollIntoView(self):
-		if not self.HTMLNode:
-			return
+	@classmethod
+	def _scrollHTMLNodeIntoView(
+			cls,
+			node: comtypes.client.lazybind.Dispatch,
+			alignToTop: bool = True
+	):
 		try:
-			self.HTMLNode.scrollInToView()
+			node.scrollIntoView(alignToTop)
 		except (COMError,NameError):
-			pass
+			log.debugWarning("HTML node scrollIntoView failed", exc_info=True)
+			raise NotImplementedError
+
+	def scrollIntoView(self):
+		if self.HTMLNode:
+			self._scrollHTMLNodeIntoView(self.HTMLNode, True)

```

Comment:
Shouldn't this line return to avoid raising `NotImplementedError` if the test succeeded?

```suggestion
			return self._scrollHTMLNodeIntoView(self.HTMLNode, True)
```